.github/workflows/aws-ecs-deploy.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
Does AWS support node 18?  The reason I ask that is the current LTS version.  I think happened in October.

![image](https://user-images.githubusercontent.com/12024120/205751500-6b3150c7-9dce-40ce-a5d1-d299c6df5f51.png)

Reply:
doesn't look like it at the moment...